olog.py

```diff
@@ -16,77 +13,159 @@ def __init__(self, url, user, password):
         user : string
         password : string
         """
-        if url.endswith('/'):
-            url = url[:-1]
-        auth = requests.auth.HTTPBasicAuth(user, password)
-        self._url = url
-        self._session = requests.Session()
-        # I have requested that working certificates be properly configured on
-        # the server. This is unfortunately a necessary workaround for now....
-        self._session.verify = False
-        self._kwargs = dict(headers=headers, auth=auth)  # for every request
-
-    def list_logbooks(self):
-        url = f'{self._url}/resources/logbooks'
-        res = self._session.get(url, **self._kwargs)
-        res.raise_for_status()
-        return res.json()
+        if url[-1] != '/':
```

Comment:
You can use [`endswith`](https://docs.python.org/3/library/stdtypes.html#str.endswith) as a more elegant solution here.